src/cmd-buildextend-metal

```diff
@@ -85,14 +85,23 @@ fi
 disk_args=()
 # SecureExecution extra stuff
 secex_image_suffix=
-secex_hostkey_drive=()
 if [[ $secure_execution -eq "1" ]]; then
     hostkey=$(realpath "$hostkey")
     secex_image_suffix="-secex"
     disk_args+=("--with-secure-execution")
-    secex_hostkey_drive=("-drive" "if=none,id=hostkey,format=raw,file=$hostkey,readonly=on" \
+    qemu_args+=("-drive" "if=none,id=hostkey,format=raw,file=$hostkey,readonly=on" \
                          "-device" "virtio-blk,serial=hostkey,drive=hostkey")
 fi
+# Add extra arguments for runvm
+qemu_args=()
```

Comment:
shouldn't this be declared before line 88 ?

Reply:
Yes, i moved it up together with `read`, so that qemu_args does not get overwritten by it